src/main/java/org/myrobotlab/service/AudioFile.java

```diff
@@ -222,20 +221,14 @@ public void playResource(String filename) {
   };
 
   public void playResource(String filename, Boolean isBlocking) {
-    log.warn("Audio File playResource not implemented yet.");
-    // FIXME AudioData needs to have an InputStream !!!
-    // cheesy - should play resource from the classpath
-    playFile(filename, isBlocking);
-    // TODO: what/who uses this? should we use the class loader to
-    // playFile(filename, isBlocking, true);
-
+    playFile(getResourceDir() + File.separator + filename, isBlocking);

```

Comment:
this changes the semantics of how this method behaves.. please add a javadoc comment to say that this will play a file relative to the resource directory .

Reply:
Sure, I added a javadoc, but I think you are confused.  The original playFile is unchanged, the reason I **_added_** this file was to make the service script which has been broken for months if not years work.  playFile exists as it has forever .. playResource I actually implemented correctly, with the correct getResourceDir path ... previously is just logged this !

log.warn("Audio File playResource not implemented yet.");

:D